src/middlewared/middlewared/api/base/types/base/string.py

```diff
@@ -57,5 +60,7 @@ def __get_pydantic_core_schema__(
 NonEmptyString = Annotated[str, Field(min_length=1)]
 LongNonEmptyString = Annotated[LongString, Field(min_length=1)]
 TimeString = Annotated[str, AfterValidator(Time())]
+NetbiosDomain = Annotated[str, AfterValidator(validate_netbios_domain)]
```

Comment:
The accepts had a length check.  Maybe...
`NetbiosDomain = Annotated[str, AfterValidator(validate_netbios_domain), Field(max_length=15)]`

Same for `NetbiosName`  (below)

However.... I'm not sure netbiosalias has the same length restriction.

Reply:
It does. I added to the base validator for it.